sdk/tables/data-tables/README.md

```diff
@@ -123,6 +123,24 @@ const { TableServiceClient, AzureNamedKeyCredential } = require("@azure/data-tab
 
 The `TableServiceClient` requires a URL to the table service and an access credential. It also optionally accepts some settings in the `options` parameter.
 
+#### `TableServiceClient` with AzureNamedKeyCredential
+
+You can instantiate a `TableServiceClient` with a `AzureNamedKeyCredential` by passing account-name and account-key as arguments. (The account-name and account-key can be obtained from the azure portal.)
+[ONLY AVAILABLE IN NODE.JS RUNTIME]
```

Comment:
NIT: The nodejs only fact was mentioned in the [authenticate a client section](https://github.com/Azure/azure-sdk-for-js/blob/b6c2de4387d8c6a374e591143dc251e3bc325e9d/sdk/tables/data-tables/README.md#authenticate-a-tableserviceclient), should it be mentioned here again?

Reply:
I like to keep it explicit here as well in case the reader jumps directly to this sample